ngraph/core/include/ngraph/util.hpp

```diff
@@ -78,6 +78,8 @@ namespace ngraph
     NGRAPH_API
     size_t hash_combine(const std::vector<size_t>& list);
     NGRAPH_API
+    size_t hash_combine(const void* v, int64_t size);
```

Comment:
I think we shouldn't add `hash_combine()` method to nGraph API. It can be a local function inside the serialization cpp file.

Reply:
Done.